client/src/main/proto/spine/base/command.proto

```diff
@@ -95,7 +95,7 @@ message CommandContext {
 // Enumeration of possible failures when validating a command.
 enum CommandValidationError {
     // Reserved value.
-    UNKNOWN = 0;
+    UNDEFINED_COMMAND_VALIDATION_ERROR = 0;
```

Comment:
The problem is that we have to name enum constants differently if they are located in one package, so I throught we can differ them this way.